test/index.js

```diff
@@ -1,8 +1,36 @@
-const expect = require('chai').expect,
+const expect = require('chai').expect;
 const toRGBAArray = require('..');
+const Canvas = require('canvas');
+const range = require('lodash.range');
+const fs = require('fs');
+const path = require('path');
 
-describe('my thing', function () {
-  it('should work', function () {
-    toRGBAArray('thing');
+describe('to-rgba-array', function () {
+  it('will work with a canvas', function () {
+    var canvas = new Canvas(200, 200);
+    return toRGBAArray(canvas).then(function (result) {
+      expect(result).to.be.a('Uint8ClampedArray');
```

Comment:
might also be worth checking the pixels.

At least the length of the array which should be `4*width*height`.

I often check the values of the things as well. Like [here](https://github.com/scijs/get-pixels/blob/master/test/test.js)

You don't have to check every pixel, but often OK to get a slice of 16 or 32 bytes and do a comparison. That way if anything changes in the future you can pick up if something has changed.